libs/generators/src/lib/generators/generators-menu/generators-menu.component.scss

```diff
@@ -0,0 +1,14 @@
+:host {
+    .container {
```

Comment:
why place it into the `:host`? It's inside the host by default

Reply:
@agroupp are you sure? is it getting encapsulated even though its outside host?

Reply:
for sure